packages/features/flags/hooks/index.ts

```diff
@@ -22,8 +22,6 @@ if (process.env.NEXT_PUBLIC_IS_E2E) {
 }
 
 export function useFlags(): Partial<AppFlags> {
-  const query = trpc.viewer.features.map.useQuery(undefined, {
-    initialData,
-  });
```

Comment:
According to Tansack, we only need to provide initialData if we actually want to see it for a given query. Otherwise, the cache entry will be filled with initialData on every new entry if we provide it>
More Context:- https://github.com/TanStack/query/discussions/5993,
https://github.com/TanStack/query/discussions/6383